app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  load_and_authorize_resource only: [:create]
+  layout false
+
+  rescue_from CanCan::AccessDenied do |exception|
```

Comment:
I thought so, but it is your code, so I left it for you to refactor it according to what's best strategy here.